### ISSUE TITLE
Schedule pipeline based on cron

### ISSUE THREAD

**Issue by aman9922** [devtron-labs/devtron#452]
# Summary

We want to run our automation suite using Devtron pipeline. For that we need pipeline to get triggered based on cron.

# Proposal
1. Create schedules based on cron.
2. Add pipeline variables so that we can run job based on pipeline variable.

**Comment by pghildiyal** [comment]
Hi Aman,

Thanks for showing interest in using Devtron for any questions please reach out to us at https://discord.com/invite/jsRG5qx2gp


Can you please elaborate the use case. So that we can prioritize the same.

Also, please check if below points help you solve your current use case
1. Devtron continuously monitors the code change and build the images based on image or tag, this allows developers  to iterate faster.
2. CI nodes are in autoscaling so that cost is incurred only when CI pipeline is running else cost is zero.
3. Deployment can be made automatic and manual, in case of manual images created will not be deployed unless triggered manually.

Regards

**Comment by aman9922** [comment]
Hi Prashant,

We are having a repo where we have written script to automate manual test cases. We want these scripts to run every day at a particular time. We have created a pipeline for this. Now we want the pipeline to execute on a particular time everyday so that we will get to know the test report and evaluate builds which are breaking the application.

We are having different applications. The script to execute test for all these application is present in one repo. We have created different jobs to test different applications in yaml file. Let say we want to run application X at 10 AM everyday. So we have to create a schedule with cron and pass one pipeline variable to run tests on application X. Based on the pipeline variable the particular job to test application X will execute. So this is the reason we need pipeline variable.

Regards
